### PR TITLE
Add VK.com Auth

### PR DESCRIPTION
**Pull request by oshliaer**
vk.com is one of the most popular social networks.

_**Be careful**. Implicit Flow's Methods are not used on the server side. They will always return "error_code": 15, "error_msg": "Access denied: no access to call this method"_

### CODE REVIEW

File: samples/VK.gs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      // Set the name of the callback function that should be invoked to complete
+      // the OAuth flow.
+      .setCallbackFunction('authCallback')
+
+      // Set the property store where authorized tokens should be persisted.
+      .setPropertyStore(PropertiesService.getUserProperties())
+
+      // Set the scope and additional specific parameters if its are supported
+      .setScope('groups,offline')
+      .setParam('approval_prompt', 'force');
```
Comment (erickoledadevrel) [inline samples/VK.gs]: This doesn't seem to be a supported parameter.

File: samples/VK.gs [hunk 2]
```diff
@@ -0,0 +1,81 @@
+/**
+ *
+ * VK's Auth flow https://vk.com/dev/authcode_flow_user
+ * Scopes list https://vk.com/dev/permissions
+ * Be careful. Implicit Flow's Methods are not used on the server side. They will always return "error_code": 15, "error_msg": "Access denied: no access to call this method"
```
Comment (erickoledadevrel) [inline samples/VK.gs]: It's not clear what this is referring to.

File: samples/VK.gs (hunk 1 above)
Comment (oshliaer) [reply samples/VK.gs]: @erickoledadevrel , my fail! Thanks God I'm not in Google! =)

**Comment by erickoledadevrel** [comment]
No worries, a couple small fixes and then I'd be happy to merge.

**Comment by oshliaer** [comment]
@erickoledadevrel , I've added the changes to my fork and the PR is updated too.

**Pull request merged**